doc/source/whatsnew/v0.23.0.txt

```diff
@@ -538,7 +538,7 @@ Reshaping
 - Bug in :func:`DataFrame.merge` in which merging using ``Index`` objects as vectors raised an Exception (:issue:`19038`)
 - Bug in :func:`DataFrame.stack`, :func:`DataFrame.unstack`, :func:`Series.unstack` which were not returning subclasses (:issue:`15563`)
 - Bug in timezone comparisons, manifesting as a conversion of the index to UTC in ``.concat()`` (:issue:`18523`)
--
+- Bug in :func:`SparseDataFrame.merge` which raises error (:issue:`13665`)
```

Comment:
This can go under enhancements, saying "Merging sparse DataFrames" is now supported.

Reply:
Oh it's an enhancement alrighty thanks for the guidance.